packages/devtools-source-map/src/source-map.js

```diff
@@ -36,7 +35,7 @@ function _resolveSourceMapURL(source: Source) {
   } else if (path.isAbsolute(sourceMapURL)) {
     // If it's an absolute path, it should be resolved relative to the
     // host of the source.
-    const { protocol = "", host = "" } = parse(url);
+    const { protocol = "", host = "" } = new URL(url);
```

Comment:
This [node.js doc][1] suggests you are meant to acquire `URL` from the `url` module, but I guess it must be a global too if this change worked in the test environment...

[1]: https://nodejs.org/api/url.html#url_the_whatwg_url_api